index.html

```diff
@@ -18,9 +18,59 @@
         rel="stylesheet">
 </head>
 
-<body><!--mobile menu-->
+<body>
+    <!--pop up window-->
+    <section class="popupsec">
+        <article id="popup" class="popup absolute">
+            <div class="popupHeader">
+                <h2>Tonic</h2>
+                <div id="closePopUp">&times;</div>
+            </div>
+          <div class="popupAfterHeader">
+            <div class="canopy">
+                    <h4 class="colorh4 margin10 padtop">CANOPY</h4>
+                <ul class="popupAfterHeader">
+                    <li class="canopylist popupFlex margin10">
+                        <span>
+                            <img width="8px" height="8px" src="images/Ellipse 12.png" alt="">
+                        </span>
+                        <p>Back End Dev</p>
+                    </li>
+                    <li class="canopylist popupFlex margin10">
+                        <span>
+                            <img width="8px" height="8px" src="images/Ellipse 12.png" alt="">
+                        </span>
+                        <p>2015</p>
+                    </li>
+                </ul>
+            </div>
+        </div>
+            <div class="scrnshot width">
+                <img class="mobile" src="images/Snapshoot Portfolio (2).png" alt="">
+                <img class="desktop" src="images/Snapshoot Portfolio - Copy.png" alt="">
+            </div>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus amet et vero architecto, dolor modi eligendi animi
+                praesentium, eum eos cumque reprehenderit dolorem voluptates optio provident nemo asperiores libero ratione.
+            </p>
+            <ul class="popupFlex">
+                <li class="languages">HTML</li>
+                <li class="languages">CSS</li>
+                <li class="languages">JavaScript</li>
+            </ul>
+            <div class="popupFlex">
+                <div class="seeproject margin10">
+                    <a title="#" href="#">see live</a>
+                </div>
+                <div class="seeproject">
+                    <a title="#" href="#">see source</a>
+                </div>
+            </div>
+        </article>
+    </section>
```

Comment:
![image](https://github.com/MPDADDY/portfolio/assets/60083437/54885339-3dd2-49e2-b6f1-7ce1ff9475b5)

- As per the project requirements details in the popup window should be generated from a Javascript file and object.
- But you have used HTML to generate the details of the popup window.
- [ ] Kindly fix this issue.